### PR TITLE
YouTube Scrapper

### PR DESCRIPTION
**Pull request by prashantgo**
Script for scrapping links from YouTube Playlist.

### CODE REVIEW

File: bin/link_finder.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
+
```
Comment (Logan1x) [inline bin/link_finder.py]: why the whole code is commented?

**Comment by Logan1x** [comment]
You should add requirement.txt files too so that beginner can first download requirement and then run the code.

**Comment by Logan1x** [comment]
same [tip](https://github.com/Logan1x/Python-Scripts/pull/42#issuecomment-337465146) for you too.

**Comment by Logan1x** [comment]
@prashantgo  and for [this](https://github.com/Logan1x/Python-Scripts/pull/41#issuecomment-337465655)?

**Comment by Logan1x** [comment]
and after adding update the readme like this.
selection_024

if you need any help post a comment here I understand that you are a new contributor😊

File: bin/link_finder.py (hunk 1 above)
Comment (prashantgo) [reply bin/link_finder.py]: It was a mistake, I have corrected it.

**Comment by prashantgo** [comment]
I am not able to update README.md, it is having some conflicts and I can't figure it out.

**Comment by Logan1x** [comment]
All you have to do for resolve conflict is this.
record5

File: bin/link_finder.py (hunk 1 above)
Comment (Logan1x) [reply bin/link_finder.py]: 👍

**Comment by Logan1x** [comment]
if you need any more help just ping here, When i did my first PR no. of conversation were 60 comment on single PR 😅 so it's okay to ask if you don't know anything.

**Comment by Logan1x** [comment]
@prashantgo  still your readme.md not arranged according to alphabetically. All the given scripts are arranged alphabetically so put your link finder in between `K & M`.

**Comment by prashantgo** [comment]
Updated the readme.md finally.

File: README.md [hunk 2]
```diff
@@ -126,6 +126,15 @@ python integrate-graph.py
 ```
 
 
+### Link Finder
```
Comment (Logan1x) [inline README.md]: Add `*` before the ### se other scripts

File: README.md [hunk 3]
```diff
@@ -126,6 +126,15 @@ python integrate-graph.py
 ```
 
 
+### Link Finder
+A selenium based script for scraping links from a YouTube Playlist.
+```bash
+pip install -r link_finder.requirements.txt
+python link_finder.py
+
```
Comment (Logan1x) [inline README.md]: close th bash script here by ` adding"```"` and  delete the 213 line.

**Comment by Logan1x** [comment]
need more changes

**Comment by Logan1x** [comment]
Sorry mistakenlly PR closed Reopened again.

**Comment by Logan1x** [comment]
lot of mistakes in readme compare to mine one check again if chages are not reflected there then unfork it and fork it again.

File: README.md [hunk 4]
```diff
@@ -204,6 +213,16 @@ python server.py
 ```
 
 
+
+* ### Integrate to find area of a graph
```
Comment (Logan1x) [inline README.md]: IDK how its here? it should also be arranged alphabetically.

**Comment by Logan1x** [comment]
I request you that unfork the repo and again fork and then add your changes. your repo is not up to date that's the only problem.

File: bin/link_finder.requirements.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+selenium==3.4.3
```
Comment (Logan1x) [inline bin/link_finder.requirements.txt]: add keras and numpy too.

**Comment by Logan1x** [comment]
@prashantgo  i am seeing three python scripts there and description for only one. check the [`file changed`](https://github.com/Logan1x/Python-Scripts/pull/41/files)

**Comment by Logan1x** [comment]
If you are doing unfork and fork thing then I would suggest close this pr and create a new one.

File: bin/link_finder.requirements.txt (hunk 5 above)
Comment (prashantgo) [reply bin/link_finder.requirements.txt]: added another file for them because they are a part of different program

Comment (Logan1x) [reply bin/link_finder.requirements.txt]: why did you never added readme description for them?

Comment (prashantgo) [reply bin/link_finder.requirements.txt]: My apologies. I have added for them now.

**Comment by Logan1x** [comment]
I understand are new to open soucre world but can you please delete/unfork repo from your profile and go to my profile and fork the repo again and put the changes you added.

**Comment by Logan1x** [comment]
close this pull request and add new one

**Comment by Logan1x** [comment]
just clone it ofline and copy paste the whole thing

**Comment by prashantgo** [comment]
Hey, I just made all the changes you told me to do. If there is a problem you should tell me.

**Pull request closed without merging**